src/launcher/index.css

```diff
@@ -0,0 +1,131 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+
+.jp-LauncherWidget {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+
+.jp-LauncherWidget-dialog {
+  padding-bottom: 12px;
+  font-size: var(--jp-ui-font-size1);
+  color: var(--jp-ui-font-color1);
+  background: var(--jp-layout-color1);
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+  box-shadow: 0 2px 7px 1px #BDBDBD;
+}
+
+
+.jp-LauncherWidget-logo {
+  flex: 0 0 50px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 100%;
+  background-color: var(--jp-layout-color1);
+  background-size: 232px 50px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-top: 20px;
+}
+
+
+.jp-LauncherWidget-subtitle {
+  background-color: var(--jp-layout-color1);
+  font-size: var(--jp-ui-font-size2);
+  width: 100%;
+  margin-top: 4px;
+  font-weight: 300;
+}
+
+.jp-LauncherWidget-header {
+  padding-top: 16px;
+  padding-bottom: 4px;
+}
+
+
+.jp-LauncherWidget-body {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+
+.jp-LauncherWidget-column {
+  padding-left: 20px;
+  padding-right: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+
+.jp-LauncherWidget-image {
+  flex: 0 0 auto;
+  min-height: 72px;
+  min-width: 56px;
+  max-width: 56px;
+  background-size: 56px 72px;
+  background-repeat: no-repeat;
+
+}
+
+.jp-LauncherWidget-image:hover {
+  cursor: pointer;
```

Comment:
This doesn't need to be in a `:hover`, it can go above in the `.jp-LauncherWidget-image` - it's understood that the cursor changes on hover.